src/date-util/date-util.spec.ts

```diff
@@ -444,4 +444,68 @@ describe('DateUtil', () => {
       expect(durationTo('09:60:50')).toEqual(0);
     });
   });
+
+  describe('fromNow', () => {
+    const fromNow = DateUtil.fromNow;
+
+    beforeAll(() => {
+      jest.useFakeTimers().setSystemTime(new Date('2020-01-01')); // 00:00:00Z
+    });
+    afterAll(() => {
+      jest.clearAllTimers();
+    });
+
+    it('should return year diff from now', () => {
```

Comment:
테스트 description에 기대하는 동작이 나오는 조건에 대해서도 같이 명시되면
테스트 본문을 모두 읽고 이해하지 않아도 이 테스트가 무엇을 위한 테스트인지 빠르게 이해하는데 더 도움이 될 것 같습니다.

아래는 예시로 하나 적어봤습니다.

```suggestion
    it('should return year diff from now if the diff is bigger than a year', () => {
```

Reply:
적용했습니다